src/cloud/env/environments-ssh.md

```diff
@@ -118,20 +118,13 @@ You need the following requirements to sFTP into cloud environments:
 *  You need to use a client that supports SSH key authentication for sFTP and use your SSH public key.
 *  Your public SSH key must be added to the target environment. For Starter environments and Pro Integration environments, you can add it through the Project Web Interface. For Pro Staging and Production, you must enter a [Support ticket]({{ site.baseurl }}/cloud/trouble/trouble.html) with your public key attached. **Never provide your private SSH key.**
 
-When configuring sFTP, use your SSH public key and the following information for access:
+When configuring sFTP, use your SSH access destination (see [SSH to an Environment](#ssh), example `<project-id-name>@<cloud-host>`) and the following information for access:
```

Comment:
There is more information in that access command, so it's nice to be clear. Unless of course, I misunderstood your intention. If it is the access command you are referring, then I suggest:

```suggestion
When configuring sFTP, use the information from your [SSH access environment command](#ssh) (`<project-id>-<environment-id>--<app-name>@ssh<cloud-host>`) and the following information:
```